src/platform/Darwin/KeyValueStoreManagerImpl.h

```diff
@@ -45,6 +45,8 @@ class KeyValueStoreManagerImpl final : public KeyValueStoreManager
     friend KeyValueStoreManagerImpl & KeyValueStoreMgrImpl();
 
     static KeyValueStoreManagerImpl sInstance;
+
+    bool mIsRunning = false;
```

Comment:
maybe this should be called `mInitialized` given the usage. `Run` sounds like it would have threads/background processing.